contents/overlay/dialogs/simple-dialog/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "en": {
+    "title": "Simple Dialog",
+    "description": "It is a simple dialog."
+  },
+  "ja": {
+    "title": "シンプルなダイアログ",
+    "description": "シンプルなダイアログです。"
+  },
+  "options": {
+    "container": {
+      "centerContent": true,
+      "p": "md"
+    }
+  },
```

Comment:
```suggestion
    },
    "iframe": true
  },
```
It would be better to set up an iframe so that it fits within the screen.

<img width="1415" alt="image" src="https://github.com/user-attachments/assets/dfd0dedb-06fb-46f9-9eaf-54cb8b0cfe35">

Reply:
Thank you for the review. I tried to apply this review, but I believe it should not be applied because the `onClose` area is limited and scrolling is blocked, which deteriorates the user experience. I also considered applying `blockScrollOnMount={false}` to `<Dialog />`, but I gave up on this method because I do not want `blockScrollOnMount={false}` to be applied when copying. Is there a way to achieve a view that can be scrolled from the beginning, like the example in https://tailwindui.com/components/application-ui/overlays/modal-dialogs?

Reply:
resolved [c626db6](https://github.com/yamada-ui/yamada-components/pull/357/commits/c626db688dfa10896d7c943fff4bdc3c3b48de88)